src/main/java/org/launchcode/liftoffrecipetracker/models/Recipe.java

```diff
@@ -0,0 +1,107 @@
+package org.launchcode.liftoffrecipetracker.models;
+
+import javax.persistence.Entity;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.HashMap;
+
+
+@Entity
+public class Recipe extends AbstractEntity {
+	//properties
+	private HashMap<String, String> ingredients;
+
+	@NotBlank(message = "Recipe name required.")
+	@NotNull
+	@Size(min=2,max=50, message = "Recipe name must be between 2 and 50 characters long.")
+	private String name;
+
+	@NotNull
+	@NotBlank
+	@Size(min=4, max=1000, message = "Directions are required.")
+	private String directions;
+
+	@NotBlank
+	@Min(value = 1, message = "Must be at least one serving.")
+	private int servings;
+
+	private int cookTime;
+	private int prepTime;
```

Comment:
Should we add some validation here? We might consider adding at least a minimum size so there aren't things like negative cooktimes.